modules/contracts/apps/payment/src/msg.rs

```diff
@@ -31,6 +32,8 @@ pub enum AppExecuteMsg {
     #[cfg_attr(feature = "interface", payable)]
     Tip {},
     UpdateConfig {
+        desired_asset: Option<AssetEntry>,
```

Comment:
There's no way to set desired_asset to None in the contract

Reply:
Will be adressed by this issue: https://linear.app/abstract-sdk/issue/ABS-269/add-clearable-type-in-abstract-sdk